ui/lib/core/addon/components/selectable-card-form.hbs

```diff
@@ -0,0 +1,30 @@
+<form class="selectable-card is-rounded no-flex" data-test-selectable-card-form={{@testLabel}}>
```

Comment:
Oh yes! I will definitely update that 😊 I was originally trying to avoid `@title` since it may have spaces in between but I just realized that shouldn't affect anything.

For things like input fields, I think it _may_ be better to have a form since it's better for accessibility. Also curious what others think as well!

Reply:
Yeah - we don't have a great pattern for this. But I'd agree! That putting all inputs in forms would be best case!